grpc/interceptors_test.go

```diff
@@ -40,6 +42,14 @@ func testInvoker(_ context.Context, method string, _, _ interface{}, _ *grpc.Cli
 	return nil
 }
 
+func testRequesterCancelledInvoker(_ context.Context, method string, _, _ interface{}, _ *grpc.ClientConn, opts ...grpc.CallOption) error {
+	if method == "-requester-cancelled" {
+		return status.Error(1, context.Canceled.Error())
+	}
```

Comment:
just add this case to `testInvoker` above, so it can fail, fail due to cancellation, or succeed.